bin/scripts/config-helper.ts

```diff
@@ -0,0 +1,157 @@
+const fs = require('fs');
+const figlet = require("figlet");
+const { prompt } = require('enquirer');
+
+
+// These properties must always be set
+let questions = [
+    {
+      type: "input",
+      name: "AWS_ACCOUNT",
+      message: "The AWS account ID for the account MLSpace will be deployed into.",
+    },
+    {
+      type: "input",
+      name: "AWS_REGION",
+      message: "The region that MLSpace resources will be deployed into.",
+    },
+    {
+        type: "input",
+        name: "OIDC_URL",
+        message: "The OIDC endpoint that will be used for MLSpace authentication",
+    },
+    {
+        type: "input",
+        name: "OIDC_CLIENT_NAME",
+        message: "The OIDC client name that should be used by MLSpace for authentication",
+    },
+]
+
+
+console.log(figlet.textSync("MLSpace Config Wizard"));
+
+createConfig();
+
+async function createConfig() {
+    const basic = "basic";
+    const advanced = "advanced";
+    const advancedOrBasicResponse = await prompt({
+        type: "select",
+        name: "configType",
+        choices: [
+            { message: "Basic Config - only prompts for the properties which must be set in order to deploy MLSpace.", name: basic },
+            { message: "Advanced Config - prompts for required fields as well as optional configurations which are commonly customized.", name: advanced },
+        ],
+        message: "Select a configuration type:",
+    })
+
+    if(advancedOrBasicResponse.configType === advanced) {
+        await advancedPrompts();
+        console.log(questions.length)
+    }
+
+
+    // The final list of questions has been created, this answer object will be the final config JSON
+    const answers = await prompt(questions);
+    
+      
+    console.log(`Response: ${JSON.stringify(answers)}`); 
```

Comment:
Is this supposed to be here?

If we are showing the provided answers we may want to say something like "Provided Configurations:" instead of "Response"

Reply:
Thanks forgot to remove; will correct in next commit